config/zones/AR.yaml

```diff
@@ -162,3 +162,12 @@ fallbackZoneMixes:
 parsers:
   production: AR.fetch_production
 timezone: America/Argentina/Buenos_Aires
+subZoneNames:
```

Comment:
I think we will not compute Argentina as an aggregated zone. As we have direct data for it, it's probably better to not have it as an aggregated zones. Though, this will require some changes on our backend to interpret it as a country view.

Reply:
Agree, that would be some changes happening in the backend, I will add a note for us there.